packages/auth/src/accessToken/service.test.ts

```diff
@@ -199,9 +201,6 @@ describe('Access Token Service', (): void => {
       token = await AccessToken.query(trx).findById(token.id)
       expect(token).toBeUndefined()
     })
-    test('Cannot revoke nonexistent token', async (): Promise<void> => {
-      expect(accessTokenService.revoke('uuid')).rejects.toBeInstanceOf(Error)
-    })
```

Comment:
Why was this removed?

Reply:
The revoke function originally called `findById` instead of `findOne` using like the `managementId` like it should be. When it was using `findById` it would error if it couldn't find it, but it doesn't if `findOne` fails to find it. I decided to not re-implement this behavior because of the following:

https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-6.2
```
If the access token has already expired or has been revoked through other means, the AS SHOULD honor the revocation request to the token management URI as valid, since the end result is still the token not being usable.